service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -26,25 +26,26 @@ For more information about upgrading Operators, refer to the xref:../../operator
 endif::[]
 
 [id="upgrading-control-plane"]
-== Upgrading the Service Mesh control plane
+== Upgrading the control plane
 
 You must manually update the control plane for minor and major releases.
 
+While you can deploy multiple versions of the control plane in the same cluster, {SMProductName} does not support canary upgrades of the service mesh.  That is, you can have different SCMP resources with different values for `spec.version`, but they cannot be managing the same mesh.
```

Comment:
```suggestion
Although you can deploy many versions of the control plane in the same cluster, {SMProductName} does not support canary upgrades of the service mesh.  That is, you can have different SCMP resources with different values for `spec.version`, but they cannot be managing the same mesh.
```

Reply:
These two changes are per the word usage list in the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage